File: shbcloud-employee/src/main/java/shb/cloud/employee/entity/ShbEmployee.java
package shb.cloud.employee.entity;

import lombok.*;
import org.springframework.data.annotation.Id;
import org.springframework.data.relational.core.mapping.Table;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Table("employee")
@ToString
public class ShbEmployee {

  @Id private String employeeId;
  private String employeeName;
  private String contactNumber;
}

File: shbcloud-employee/src/main/kotlin/shb/cloud/employee/ShbEmployee.kt
package shb.cloud.employee

import lombok.Getter
import lombok.Setter
import org.springframework.data.annotation.Id
import org.springframework.data.relational.core.mapping.Table

@Getter
@Setter
@Table("employee")
class ShbEmployee {

    @Id
    var employeeId: String = "sampleId";
    var employeeName: String = "sampleName";
    var contactNumber: String = "010-0000-0000";

}